resources/scenes/game_scenes/level_1.tres

```diff
@@ -6,6 +6,6 @@
 [resource]
 script = ExtResource("2_mss3e")
 scene = ExtResource("1_ee3oc")
-scene_id = "levelTest"
-scene_is_pausable = true
+scene_id = "game_world"
+scene_is_pausable = false
```

Comment:
why this change? don't we want to be able to pause?

Reply:
accidental, fixed now